src/components/movieCardFilter/movieCardFilter.js

```diff
@@ -0,0 +1,39 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types'
+
+import './movieCardFilter.css'
+
+const MovieCardFilter = (props) => {
+
+  const buttons = [
```

Comment:
all functionality/variables that don't use a component props/state should be moved out of the component

Reply:
done